### PR TITLE
C# pages: Added .NET Core

### PR DESCRIPTION
**Pull request by RheaAyase**

### CODE REVIEW

**Comment by RheaAyase** [comment]
Hey @pvalena, is this one okay or am I still missing something? =)

**Comment by pvalena** [comment]
> Hey @pvalena, is this one okay or am I still missing something? =)

Hello. I am afraid I am out of time to review this for now, because other high-priority work is blocking me. Some other volunteer has to take a look at this or it'll have to wait a bit.

There has also been a website update on Mon/Tue, so I do not expect another one to happen for 2-3 weeks. (This can change with some significant content to push.)

**Comment by RheaAyase** [comment]
Guys, I need this in for conference on Wednesday, it was idling here for long enough. Could we do something about it please, asap?

**Comment by RheaAyase** [comment]
@pvalena or I don't know who would be the go-to person. Please, this needs to go out before 18th July

File: tech/languages/csharp/csharp.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: C# - IDEs
+subsection: csharp
+section: tech-languages
+order: 1
+version: 7.8.4
+description: "C# programming language, .NET Core and Mono open source frameworks."
+---
+
+# C# IDE's in Fedora
```
Comment (Conan-Kudo) [inline tech/languages/csharp/csharp.md]: `IDEs`

File: tech/languages/csharp/csharp.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## JetBrains Rider
+
+_.NET Core and Mono_
+
+JetBrains Rider is full featured IDE for all things C#, and even F# on .NET Core. If you are familiar with other JetBrains products, such as the IntelliJ or Resharper, you might like this one as well.
+
+[jetbrains.com/rider](http://jetbrains.com/rider)
+
+## Mono develop
```
Comment (Conan-Kudo) [inline tech/languages/csharp/csharp.md]: `MonoDevelop`

File: tech/languages/csharp/csharp.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+subsection: csharp
+section: tech-languages
+order: 1
+version: 7.8.4
+description: "C# programming language, .NET Core and Mono open source frameworks."
+---
+
+# C# IDE's in Fedora
+
+## VisualStudio Code
```
Comment (Conan-Kudo) [inline tech/languages/csharp/csharp.md]: `Visual Studio`

**Comment by pvalena** [comment]
@Conan-Kudo 

> The C# IDEs in Fedora section seems wrong, since it refers to IDEs that we don't have and can't have in Fedora repositories. I'd recommend dropping those and focusing on the things we do have in our repositories.

This is not wrong at all as Fedora Developer Portal is intended for developers on Fedora, not necesarilly developers of Fedora or using only resources from Fedora(see our [wiki pages](https://fedoraproject.org/wiki/Websites/Developer)).
IOW it's fine here to refer to upstream etc. as long as it works on latest Fedora stable ([Fedora specific](http://developer-portal.github.io/contributing/write-introduction.html) etc.).

**Comment by pvalena** [comment]
Thanks for joining the review procces!

**Comment by pvalena** [comment]
`Replaced HTML with Markdown` looks cool.
Please replace the `**headlines**` from the same file with markdown ones too.

**Comment by pvalena** [comment]
On a second look(on `tech/languages/csharp/dotnet-usage.md`) I wonder if this is a correct place for man page or `--help` kind of content, as this should be more of a guide or reference.
Documentation itself should be located elsewhere. It's however cool to point to it and/or give some frequent usage examples.

**Comment by RheaAyase** [comment]
@pvalena yes that is correct, and ideally this page would first explain the difference between e.g. publish and build, which is confusing, guide a user through some examples, and only then provide detailed explanation such as the current state of the page here. Sadly I don't think that we've got time to do that right now... We could exclude this page from "going live" and add it in later, or release it as-is with further improvement within the next two weeks.

**Comment by pvalena** [comment]
> We could exclude this page from "going live" and add it in later, or release it as-is with further improvement within the next two weeks.

Yes, I think that would be best.

File: tech/languages/csharp/dotnet-installation.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+# .NET SDK installation
+
+.NET Core is still work in progress, and below download locations contain packages that are not entirely _free._ Please be patient, we should have reasonable packages within a few months.
+
+Choose based on your Fedora and SDK version:
+
+* [Microsoft's 2.0 preview1](https://www.microsoft.com/net/core/preview#linuxfedora) (F25 `.csproj`)
+* [Microsoft's 1.X packages](https://www.microsoft.com/net/download/linux) (F23 & 24 `.json`)
+* [Fedora .NET SIG packages](https://copr.fedorainfracloud.org/coprs/nmilosev/dotnet-sig) (F25+ `.json`)
```
Comment (pvalena) [inline tech/languages/csharp/dotnet-installation.md]: There are references to F23, 24, 25, but on FDP we intend to support only latest stable Fedora (currently F26).

Comment (RheaAyase) [inline tech/languages/csharp/dotnet-installation.md]: This is due to Microsoft releasing anything and everything a year late. e.g. they released the official 1.x package for F23 a month before it was eol... (iirc)

I removed F23 reference, we will replace all of these weird packages once we get ours out - working on the build now, we should have `.NET Core 2.0 Preview2` released within a week or two - for 25 and 26 (hopefully we won't hit any major issues on 26...)

**Comment by pvalena** [comment]
LGTM.

**Pull request merged**